airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/data/AirbyteSchemaIdentityMapper.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cdk.load.data
+
+open class AirbyteSchemaIdentityMapper : AirbyteSchemaMapper {
```

Comment:
upon further thought: is there even a reason to have this class? I.e. why not just put all the identity functions directly into the AirbyteSchemaMapper interface?